Modules/Payments/Tests/Unit/PaymentLogServiceTest.php

```
@@ -0,0 +1,107 @@
+        PaymentLog::factory()->count(3)->create([
+            'invoice_id' => $invoice->invoice_id,
+            'payment_log_date' => now()->subDays(1),
+        ]);
+
+        /** Act */
+        $result = $this->service->getAllWithRelations();
+
+        /** Assert */
+        $this->assertGreaterThanOrEqual(3, $result->total());
+        $this->assertTrue($result->first()->relationLoaded('invoice'));
+    }
```

PHPMD (2.15.0)

The method it_gets_all_payment_logs_with_relations_paginated is not named in camelCase. (undefined)

(CamelCaseMethodName)